docs/release-notes/NuGet-6.12.md

```diff
@@ -24,6 +24,14 @@ NuGet distribution vehicles:
 * VS PM UI shows warning icon about package vulnerability even after upgrade [#13866](https://github.com/NuGet/Home/issues/13866)
 * dotnet nuget why reports missing argument, even though it ran [#13908](https://github.com/NuGet/Home/issues/13908)
 
+## Summary: What's New in 6.12.3
+
+NuGet 6.12.3 is available in Visual Studio 17.12.4.
```

Comment:
Has this been inserted into VS already? I didn't think it has been, so I don't think we can be confident that the fix will be released in 17.12.4 yet. It might slip to .5

Reply:
It'll be 17.12.5